module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -3,9 +3,44 @@
 int NWD(int lhs, int rhs) {
     // TODO: Implement me :)
     return -1;
+
+    if (rhs == 0){
+        return lhs;
+    }
+    if (lhs == 0){
+        return rhs;
+    }
+
+    if (lhs < 0){
+        lhs = -1*lhs;
+    }
+    if (rhs < 0){
+        rhs = -1*rhs;
+    }
+
+    int divider = (lhs > rhs) ? lhs%rhs : rhs%lhs;
+
+    if (divider == 0)
+        return (lhs>rhs) ? rhs : lhs;
+    else
+        return (lhs > rhs) ? NWD(divider, rhs) : NWD (divider, lhs);
```

Comment:
I saw that you are using brackets after `if` even if it has one line, so please add them here too.
```suggestion
    if (divider == 0) {
        return (lhs>rhs) ? rhs : lhs;
   } else {
        return (lhs > rhs) ? NWD(divider, rhs) : NWD (divider, lhs);
   }
```